recipes/rte-rrtmgp/meta.yaml

```diff
@@ -0,0 +1,75 @@
+{% set name = "rte-rrtmgp" %}
+{% set version = "v1.9.0" %}
+{% set run_validation_plots = false %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  url: https://github.com/earth-system-radiation/rte-rrtmgp/archive/refs/tags/v1.9.zip
+  sha256: 7d61d67b562958b4b6824196ac27673e0ecde297878cb948de647d6e4f70bcc8
+
+build:
+  number: 0
+  script_env:
+    - RUN_VALIDATION_PLOTS={{ run_validation_plots }}
```

Comment:
We discourage the use of custom jinja b/c we are moving to recipe v1, where jinja won't be allowed. With that said, you don't have to change it here now, just keep it in mind moving forward.

Reply:
Could you please help us understand what this is being used for? Is this a test? Is this meant for end users?